atom/browser/atom_browser_client.cc

```diff
@@ -90,14 +98,22 @@ bool AtomBrowserClient::ShouldCreateNewSiteInstance(
     content::BrowserContext* browser_context,
     content::SiteInstance* current_instance,
     const GURL& url) {
-
+  if (g_suppress_renderer_process_restart) {
```

Comment:
>  that's why I did workaround with g_reuse_renderer_process_for_new_window variable and prevented renderer process spawn in ShouldCreateNewSiteInstance.

The render process is not reused for same origin child window, `process.pid` returns different values.